### ISSUE TITLE
Error in Run model and Results display

### ISSUE THREAD

**Issue by araikes** [NISOx-BDI/SwE-toolbox#184]
Good morning @BryanGuillaume and @nicholst,

I am attempting to run a longitudinal VBM analysis in SwE 2.2.1 (downloaded it yesterday). I'm running MATLAB 2020a and SPM12 7771.

I've created a model and when I go to run the model (`Batch -> SwE -> Run model`), I get the error at the bottom.

`spm_matlab_version_chk` is deprecated in the current version. If I comment out the version check, Run model completes. The same issue happens with `Results display` as `spm_matlab_version_check` is present in `swe_contrasts.m` Likewise, I can comment out the check and it completes. 

Just thought you'd like the heads up. 

```
Initialising parameters                 :                        ...done
Output images                           :                 ...initialised
Chunk   1/1                             :                        ...done
Saving SwE.mat                          :                     ...writing15-Dec-2020 10:38:02 - Failed  'Run Model'
Undefined function 'spm_matlab_version_chk' for input arguments of type 'char'.
In file "/groups/adamraikes/matlab_toolboxes/spm12/toolbox/SwE/swe_cp.m" (???), function "swe_cp" at line 1184.
In file "/groups/adamraikes/matlab_toolboxes/spm12/toolbox/SwE/swe_run_rmodel.m" (???), function "swe_run_rmodel" at line 34.

The following modules did not run:
Failed: Run Model
```

**Comment by nicholst** [comment]
Thanks!

I am aware of the problem and fixed it with this PR:
https://github.com/NISOx-BDI/SwE-toolbox/issues/165

I'll make a new release to incorporate this.

**Comment by araikes** [comment]
Hi @nicholst,
Despite looking through those closed issues, I didn't see it. Sorry for the redundancy.

Thanks

**Comment by nicholst** [comment]
Now addressed in [v2.2.2](https://github.com/NISOx-BDI/SwE-toolbox/releases/tag/v2.2.2).
